contrib/inventory/ec2.ini

```diff
@@ -143,3 +143,7 @@ group_by_elasticache_replication_group = True
 # A boto configuration profile may be used to separate out credentials
 # see http://boto.readthedocs.org/en/latest/boto_config_tut.html
 # boto_profile = some-boto-profile-name
+
+# If set to true, any tag of the form "a,b,c" is expanded into a list
```

Comment:
I'd move this up to somewhere between lines 97 and 117, which is where the current group creation bit is.  Everything after is filtering.